metals/src/main/scala/scala/meta/internal/metals/FileSystemSemanticdbs.scala

```diff
@@ -22,6 +22,7 @@ final class FileSystemSemanticdbs(
 ) extends Semanticdbs {
 
   override def textDocument(file: AbsolutePath): TextDocumentLookup = {
+    // TODO need to handle java targets?
```

Comment:
This should be possible via plugin recently released that can produce semanticdb for Java files: https://github.com/sourcegraph/lsif-java

We would probably need to merge it all together for it to make sense.